framework/src/modules/pos/schemas.ts

```diff
@@ -133,22 +134,30 @@ export const configSchema = {
 		factorSelfStakes: {
 			type: 'integer',
 			format: 'uint32',
+			minimum: 1,
 		},
 		maxLengthName: {
 			type: 'integer',
 			format: 'uint32',
+			minimum: 10,
+			maximum: 30,
 		},
 		maxNumberSentStakes: {
 			type: 'integer',
 			format: 'uint32',
+			minimum: 1,
+			maximum: 20,
 		},
 		maxNumberPendingUnlocks: {
 			type: 'integer',
 			format: 'uint32',
+			minimum: 1,
+			maximum: 40,
 		},
 		failSafeMissedBlocks: {
 			type: 'integer',
 			format: 'uint32',
```

Comment:
I believe this should be `uint64`

Reply:
@gkoumout I see that in the param spreadsheet you defined `FAIL_SAFE_MISSED_BLOCKS` to have max value `MAX_UINT_64`.

While [LIP-0057](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants) defines it as `uint32`.

Which one should be corrected?

Reply:
I guess uint32 is the right thing. We use uint64 only for constants that correspond to tokens, e.g., fee, validator weight, etc, the rest is all uint32 I think.